Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use ARG to define the node version
+ARG NODE_VERSION=18
+ARG NODE_ENV=production
+
+# Use the Node.js version specified by NODE_VERSION
+FROM node:${NODE_VERSION}-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci
+
+# Copy the rest of the application code
+COPY . .
+
+# Build the application
+RUN npm run build
+
+# Expose the port the app runs on
+EXPOSE 3000
```

Comment:
Hard-coded port. I cannot start the mock warden on a custom PORT such as `5566`, I face `dial tcp 172.19.0.2:5566: connect: connection refused`.

Reply:
This should be fixed with newer commits

Reply:
No, you need to use the bridge port, not the container's port. By default, it should be `3000`. For example, if your docker-compose.yml contains:

```
...
- "${PORT:-3000}:3000"
...
```

And if `PORT` is set to `3001`, you should use `3001`, not `3000`. Did you test like that @necipallef ?

Reply:
I ran `APP_PORT=5566 docker-compose up -d --build` is this wrong?